src/sqlfluff/core/default_config.cfg

```
@@ -92,6 +92,7 @@ ignore_words = None
 [sqlfluff:rules:L016]
 # Line length
 ignore_comment_lines = False
+ignore_comment_clause = False
```

Make this consistent with config above

```suggestion
ignore_comment_clauses = False
```